server/index.js

```diff
@@ -1,16 +1,18 @@
 const express = require('express');
 const db = require('../database/index.js');
 const path = require('path');
+const cors = require('cors');
 
 const app = express();
 const port = 6070;
 
+app.use(cors());
 app.use(express.json());
 app.use(express.static(path.join(__dirname, '../client/dist')));
 app.use('/:id', express.static(path.join(__dirname, '../client/dist')));
 
 
-app.get('/api/tips/:id', (req, res) => {
+app.get(`http://localhost:${port}/api/tips/:id`, (req, res) => {
```

Comment:
@meredith-myers finally came around to review this and I think this is why your module isn't fetching the data anymore. These routes were fine as they were, you need to add http://localhost:6070 to your App.jsx file, not here 🙂